variables.tf

```diff
@@ -367,3 +346,33 @@ variable "ssm_tag_patch_window" {
   default     = "ITOPS-Wave1-Non-Mktplc-DevTestImpl-MW"
   description = "SSM Patching window for instances. For more information: https://cloud.cms.gov/patching-prerequisites"
 }
+
+variable "ami_owner_override" {
+  type        = list(string)
+  default     = [""]
+  description = "AMI owner override"
+}
+
+variable "use_bottlerocket" {
+  type        = bool
+  default     = false
+  description = "Use Bottlerocket"
+}
+
+variable "node_labels" {
+  description = "The labels to apply to the EKS nodes"
+  type        = map(string)
+  default     = {}
+}
+
+variable "node_taints" {
+  description = "The taints to apply to the EKS nodes"
+  type        = map(string)
+  default     = {}
+}
```

Comment:
These should be node-level configs.  Taints especially will break our functionality.